compiler/plugins/target/AMD-AIE/air/TransformPasses.cpp

```diff
@@ -25,5 +25,6 @@ void registerAIRTransformPasses() {
   registerAIRHerdPlacementPass();
   registerAIRSpecializeDmaBroadcast();
   registerAIRUnrollLoopForPipeliningPattern();
+  registerAIRCollapseHerdPass();
```

Comment:
A new pass which collapses 2d air herds to occupy a single column. An optimization in the long term, but a necessity in the short term given the recent IPU driver bug rendering only one column usable (temporary).